pkg/acctest/test.go

```diff
@@ -42,30 +42,33 @@ var testAccProviderConfigure sync.Once
 
 // ProtoV5MuxProviderFactories is used to instantiate both SDK v2 and Framework providers
 // during acceptance tests. Use it only if you need to combine resources from SDK v2 and the Framework in the same test.
-var ProtoV5MuxProviderFactories = map[string]func() (tfprotov5.ProviderServer, error){
-	"artifactory": func() (tfprotov5.ProviderServer, error) {
-		ctx := context.Background()
-		providers := []func() tfprotov5.ProviderServer{
-			providerserver.NewProtocol5(provider.Framework()()), // terraform-plugin-framework provider
-			provider.SdkV2().GRPCProvider,                       // terraform-plugin-sdk provider
-		}
-
-		muxServer, err := tf5muxserver.NewMuxServer(ctx, providers...)
-
-		if err != nil {
-			return nil, err
-		}
-
-		return muxServer.ProviderServer(), nil
-	},
-}
+var ProtoV5MuxProviderFactories map[string]func() (tfprotov5.ProviderServer, error)
 
 func init() {
 	Provider = provider.SdkV2()
 
 	ProviderFactories = map[string]func() (*schema.Provider, error){
 		"artifactory": func() (*schema.Provider, error) { return provider.SdkV2(), nil },
```

Comment:
Can we re-use the variable `Provider` here in place of `provider.SdkV2()`? Same for line #59.

I suspect some of the extraneous API calls (version check, call home, etc.) are from these different copies of the provider being initialized.

Reply:
Yes, we sure can.